<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEmployee_Dialog</class>
 <widget class="QDialog" name="AddEmployee_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>90</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>添加职工</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberlineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>186</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_number">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>工 号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>职 称</string>
   </property>
  </widget>
  <widget class="QComboBox" name="position_comboBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>89</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>教授</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>副教授</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>讲师</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>助教</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>72</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>入职时间</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="hireDate_dateEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>125</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>姓 名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="namelineEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="address_lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>193</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>36</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>住址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>72</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>住房面积</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>300</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>平方米</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="area_doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>300</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_next">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Exit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>370</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>年 龄</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>岁</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="age_spinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>53</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>学 位</string>
   </property>
  </widget>
  <widget class="QComboBox" name="degree_comboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="currentText">
    <string>高中</string>
   </property>
   <item>
    <property name="text">
     <string>高中</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>本科</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>硕士</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>博士</string>
    </property>
   </item>
  </widget>
 </widget>
 <tabstops>
  <tabstop>numberlineEdit</tabstop>
  <tabstop>age_spinBox</tabstop>
  <tabstop>hireDate_dateEdit</tabstop>
  <tabstop>degree_comboBox</tabstop>
  <tabstop>pushButton_next</tabstop>
  <tabstop>pushButton_Exit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Exit</sender>
   <signal>clicked()</signal>
   <receiver>AddEmployee_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
